reporters/ClickhouseReporter.js

```diff
@@ -0,0 +1,217 @@
+const {ClickHouse} = require('clickhouse');
+const os = require('os');
+const BaseReporter = require('./BaseReporter');
+const {createUrlHash} = require('../helpers/hash');
+
+// eslint-disable-next-line no-process-env
+const CLICKHOUSE_SERVER = process.env.CLICKHOUSE_SERVER || 'va-clickhouse1';
```

Comment:
I'd like to avoid those env settings in the future, but for now there is no other way of passing settings to reporter so I think it's OK. FYI I have a plan to allow passing config to reporters and collectors - #60 .

Reply:
Yeah, this is a bit of a hack as I didn't have a good way to pass options in here. I wonder if it would be easiest to just give every collector and reporter access to the `program` object to define their own options and read them back.